packages/manager/modules/pci/src/projects/new/payment/components/add/index.html

```diff
@@ -0,0 +1,48 @@
+<div class="pci-project-new-payment-method-add">
+    <div
+        class="row"
+        data-ng-repeat="authorizedPaymentMethods in $ctrl.authorizedPaymentMethods.chunks track by $index"
+    >
+        <div
+            class="col-md-6 mb-3"
+            data-ng-repeat="authorizedPaymentMethod in authorizedPaymentMethods track by $index"
+        >
+            <oui-radio
+                class="oui-radio_btn"
+                data-name="paymentType"
+                data-thumbnail
+                data-variant="light"
+                data-model="$ctrl.model.paymentMethod"
+                data-value="authorizedPaymentMethod"
+                data-on-change="$ctrl.onPaymentTypeRadioChange()"
+                data-disabled="$ctrl.globalLoading.finalize"
+            >
+                <span
+                    data-ng-bind="$ctrl.getPaymentMethodTypeText(authorizedPaymentMethod)"
+                ></span>
+            </oui-radio>
+        </div>
+    </div>
+
+    <div data-ng-if="$ctrl.model.paymentMethod.paymentType !== 'CREDIT'">
```

Comment:
Maybe some utilities function like `isCredit()`, `isBankAccount()` in the `PaymentMethod` model could be useful to avoid using constants in the view

Reply:
This is originally a simple move of the code in different files but yes you are right this is more readable and easier